### PR TITLE
Add radiology and lab combos and fix plan mistake

### PR DESCRIPTION
**Pull request by MiguelAHPpih**

### CODE REVIEW

File: configuration/pih/htmlforms/outpatientConsult.xml [hunk 1]
```diff
@@ -419,45 +421,58 @@
...
-                    <render labName="SES Lab:Exam_016"/>
-                    <render labName="SES Lab:Exam_017"/>
-                    <render labName="SES Lab:Exam_018"/>
-                    <render labName="SES Lab:Exam_019"/>
-                    <render labName="SES Lab:Exam_020"/>
-                    <render labName="SES Lab:Exam_021"/>
-                </repeat>
+                <div id="image-section">
+                    <repeat with="['1'],['2'],['3'],['4'],['5'],['6'],['7'],['8']">
+                        <span id="image-{0}" class="image">
```
Comment (brandones) [inline configuration/pih/htmlforms/outpatientConsult.xml:453]: Neither `image`, `image-field`, nor `image-name` appear to be defined in CSS anywhere.

Comment (MiguelAHPpih) [reply configuration/pih/htmlforms/outpatientConsult.xml:453]: I think it was for the expandable section js query.

Comment (brandones) [reply configuration/pih/htmlforms/outpatientConsult.xml:453]: It doesn't refer to classes at all. See the docstring: https://github.com/PIH/openmrs-config-pihemr/blob/master/configuration/pih/scripts/global/expandable.js

Comment (MiguelAHPpih) [reply configuration/pih/htmlforms/outpatientConsult.xml]: Done! @brandones

**Review by brandones — APPROVED** [review]

**Pull request merged**